### DISCUSSION TITLE
Sub-folders set as Read-only?

### DISCUSSION THREAD

**Discussion by Grabstertv** [MobiusSync/MobiusSync#65] [General]
I use Mobius for syncing my markdown notes with my laptop. For some reason, subfolders are set as readonly and i cant edit the files inside of them. Why is that and how can i make them read & write?

**Comment by 497e0bdf29873** [comment]
May I ask, which markdown editor you are using that allows editing notes in arbitrary folder? (Would be nice to have one with KaTeX support.)
